From ce277612b1e262a84a1a97e5084571b66b7dd573 Mon Sep 17 00:00:00 2001
From: Dan Callaghan <dcallagh@redhat.com>
Date: Thu, 11 Feb 2016 10:29:02 +1000
Subject: [PATCH] switch requirement from pykerberos back to kerberos

For now, Fedora still has the original "kerberos" module, not the newer
fork "pykerberos". That will work fine too.
---
 requirements.txt | 2 +-
 setup.py         | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/requirements.txt b/requirements.txt
index c49add0..98fcbbc 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -1,6 +1,6 @@
 requests>=1.1.0
 winkerberos >= 0.5.0; sys.platform == 'win32'
-pykerberos >= 1.1.8, < 2.0.0; sys.platform != 'win32'
+kerberos; sys.platform != 'win32'
 cryptography>=1.3
 cryptography>=1.3; python_version!="3.3"
 cryptography>=1.3, <2; python_version=="3.3"
diff --git a/setup.py b/setup.py
index 8381ac7..53dafc0 100755
--- a/setup.py
+++ b/setup.py
@@ -54,7 +54,7 @@ def get_version():
     ],
     extras_require={
         ':sys_platform=="win32"': ['winkerberos>=0.5.0'],
-        ':sys_platform!="win32"': ['pykerberos>=1.1.8,<2.0.0'],
+        ':sys_platform!="win32"': ['kerberos'],
     },
     test_suite='test_requests_kerberos',
     tests_require=['mock'],
-- 
2.13.6

